underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -208,7 +208,13 @@ public boolean delete(String path, boolean recursive) throws IOException {
   public boolean exists(String path) throws IOException {
     // To get better performance Swift driver does not create a _temporary folder.
     // This optimization should be hidden from Spark, therefore exists _temporary will return true.
-    return path.endsWith("_temporary") || isFile(path) || isDirectory(path);
+    if (path.endsWith("_temporary")) {
+      return true;
+    }
+    String strippedPath = stripPrefixIfPresent(path);
+    Container c = mAccount.getContainer(mContainerName);
```

Comment:
NIT: `c` -> `container` ?
and `so` -> `object` or `storedObject`,
to be more consistent with our var naming style